test_util/test_upgrade_vpc.py

```diff
@@ -37,19 +38,45 @@
 
 import test_util.aws
 import test_util.cluster
-
+from test_util.cluster_api import ClusterApi
+from test_util.helpers import CI_AUTH_JSON, DcosUser
+from test_util.marathon import get_test_app_in_docker
 
 logging.basicConfig(format='[%(asctime)s|%(name)s|%(levelname)s]: %(message)s', level=logging.DEBUG)
 log = logging.getLogger(__name__)
 
 
+def get_task_info(apps, tasks):
+    idx = 0    # We have a single app and a task for this test.
+
+    try:
+        tasks_state = tasks["tasks"][idx]["state"]
+        health_check_interval = \
+            apps["apps"][idx]["healthChecks"][idx]["intervalSeconds"]
+        idx = tasks["tasks"][idx]["id"]
+        last_success = tasks["tasks"][idx]["healthCheckResults"][idx]["lastSuccess"]
+    except IndexError as e:
+        logging.debug("Failed to get task detail: {exp}".format(exp=str(e)))
+        return None
+
+    TaskInfo = collections.namedtuple("TaskInfo", "state id health_check_interval last_success_time")
+
+    task_info = TaskInfo(
+        state=tasks_state,
+        id=idx,
+        health_check_interval=datetime.timedelta(seconds=health_check_interval),
+        last_success_time=(datetime.datetime.strptime(last_success, "%Y-%m-%dT%H:%M:%S.%fZ")))
+
+    return task_info
+
+
 def main():
     num_masters = int(os.getenv('MASTERS', '3'))
     num_agents = int(os.getenv('AGENTS', '2'))
     num_public_agents = int(os.getenv('PUBLIC_AGENTS', '1'))
     stack_name = 'upgrade-test-' + ''.join(random.choice(string.ascii_uppercase + string.digits) for _ in range(10))
 
-    pytest_cmd = os.getenv('DCOS_PYTEST_CMD', 'py.test -vv -s -rs ' + os.getenv('CI_FLAGS', ''))
```

Comment:
This space is definitely required. Otherwise you get py.test --v -s -rs--teamcity which will break